docs/visualize/tsvb.asciidoc

```diff
@@ -0,0 +1,115 @@
+[[TSVB]]
+== Visualizing your data with TSVB
+
+TSVB is a time series data visualizer that allows you to use the full power of the 
+Elasticsearch aggregation framework. With TSVB, combine an infinite 
+number of aggregations and pipeline aggregations to display complex data in a 
+meaningful way.
+
+NOTE: In Elasticsearch version 7.3.0 and later, the time series data visualizer is now referred to simply as TSVB instead of Time Series Visual Builder.
+
+TSVB comes with these types of visualizations:
+
+Time Series:: A histogram visualization that supports area, line, bar, and steps along with multiple y-axis.
+
+[role="screenshot"]
+image:images/tsvb-screenshot.png["Time series visualization"]
+
+Metric:: Displays the latest number in a data series.
+
+[role="screenshot"]
+image:images/tsvb-metric.png["Metric visualization"]
+
+Top N:: A horizontal bar chart where the y-axis is based on a series of metrics, and the x-axis is the latest value in the series.
+
+[role="screenshot"]
+image:images/tsvb-top-n.png["Top N visualization"]
+
+Gauge:: A single value gauge visualization based on the latest value in a series.
+
+[role="screenshot"]
+image:images/tsvb-gauge.png["Gauge visualization"]
+
+Markdown:: Edit the data using using Markdown text and Mustache template syntax.
+
+[role="screenshot"]
+image:images/tsvb-markdown.png["Markdown visualization"]
+
+Table:: Display data from multiple time series by defining the field group to show in the rows, and the columns of data to display.
+
+[role="screenshot"]
+image:images/tsvb-table.png["Table visualization"]
+
+[float]
+[[tsvb-data-series-options]]
+=== Configure the data series
+
+Add multiple data series with multiple aggregations composed together to create a single metric. The options available are dependent on the visualization.
```

Comment:
This sentence is a little difficult to read.  Can you remove "composed together"